sql/schema.sql

```diff
@@ -25,6 +25,13 @@ CREATE TABLE subscriber(
   PRIMARY KEY(uid)
 );
 
+CREATE TABLE discord(
```

Comment:
any more thoughts on making this a generic link table that can support linking other accounts?

Reply:
Sounds finde to me. Do you have any naming suggestion for the table?